lib/tasks/export_tasks.rake

```diff
@@ -0,0 +1,241 @@
+#
+# export_tasks.rake is a debugging tool to extract tasks from the
+# current foreman instance.
+# 
+# Run "foreman-rake export_tasks" to export tasks whcih are not listed
+# as successful. 
+# To export all tasks "foreman-rake export_tasks tasks=all"
+
+desc 'Export Dynflow Tasks'
+  task :export_tasks => :environment do
```

Comment:
the indentation is at 2, whic his hte standard. yes?
Also.. on the namespace, do you want the file moved to that directory as well?